api/redhatopenshift/resource-manager/Microsoft.RedHatOpenshift/preview/2024-06-10-preview/examples/HcpOpenShiftClusters_CreateOrUpdate_MaximumSet_Gen.json

```diff
@@ -268,30 +268,30 @@
           }
         },
         "identity": {
-          "principalId": "xlswu",
-          "tenantId": "xfqisd",
+          "principalId": "wjl",
+          "tenantId": "lgezjb",
           "type": "None",
           "userAssignedIdentities": {
-            "key4794": {
-              "principalId": "uctdckatfraombzrbkdltewc",
-              "clientId": "auud"
+            "key8996": {
+              "principalId": "ykggzlpmwfdfatpxzujiqutb",
+              "clientId": "lsgvomatprygzx"
             }
           }
         },
         "tags": {
-          "key4181": "leaswtidajsjtgmqawhdl"
+          "key691": "ovhepdiaji"
         },
-        "location": "ayecbdqonsqfowbq",
-        "id": "xioeiro",
-        "name": "vuwzuwooutjavgdhoatz",
-        "type": "utiyj",
+        "location": "ncui",
+        "id": "abdtuqlecqhqunqajf",
+        "name": "picbfltf",
+        "type": "ixbmldoscgjnp",
         "systemData": {
-          "createdBy": "lsrkqcuijqfp",
+          "createdBy": "cfriivusshykeljus",
           "createdByType": "User",
-          "createdAt": "2024-03-27T14:57:32.578Z",
-          "lastModifiedBy": "tgpmwu",
+          "createdAt": "2024-03-28T11:00:21.485Z",
+          "lastModifiedBy": "rbuvjbcbezqs",
           "lastModifiedByType": "User",
-          "lastModifiedAt": "2024-03-27T14:57:32.578Z"
```

Comment:
this regeneration of the examples might be a problem down the line when we want to check for any new generation from checkedin ungenerated changes. Is it possible to not regenerate already generated examples?

Reply:
I checked the generator and unfortunately no. The only solution that I can come up with, is to generate them and manually discard existing ones. 

Which is doable. I will make it work and document the process.